src/setup/cortx_deploy

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+# Log Levels:
+DEBUG=1
+INFO=2
+WARNING=3
+ERROR=4
+FATAL=5
+DEFAULT_LOG_LEVEL=${DEFAULT_LOG_LEVEL:-$INFO}
+
+function log {
+    LOG_LEVEL=${1:-$DEFAULT_LOG_LEVEL}
+    shift
+    [[ $LOG_LEVEL -ge $DEFAULT_LOG_LEVEL ]] && echo "log: $*"
+}
+
+function show_usage {
+    echo -e "usage: $(basename $0) -n NODE-ID [-f SOLUTION-CONFIG] [-d LEVEL][-o CORTX-CONFIG-URL]"
+    echo -e "Where:"
+    echo -e " NODE-ID         : Unique Node ID"
+    echo -e " SOLUTION-CONFIG : Solution-Config directory containing -\
+                                config.yaml, cluster.yaml and secrets.yaml"
+    echo -e "                   (Default: /etc/cortx/solution)"
+    echo -e " LOG LEVEL       : Log Level. Default 2"
+    echo -e "                   Supported log levels: DEBUG=1 INFO=2"
+    echo -e "                   WARNING=3 ERROR=4 FATAL=5"
+    echo -e " CORTX-CONFIG-URL: URL of CORTX Configuration."
+    echo -e "                   Default is yaml:///etc/cortx/cluster.conf"
+    exit 1
+}
+
+function validate_config {
+    local config=$1
+    if [ ! -f $config ]; then
+        echo -e "ERROR: Config $config not found"
+    fi
+}
+
+function yaml_config {
+    local config=$1 
+    echo "yaml://$config"
+}
+
+# Constants
+NODE_ID=0
+SETUP_ARGS=""
+SOLUTION_CONFIG="/etc/cortx/solution"
+CORTX_CONFSTORE="/etc/cortx/cluster.conf"
+while [ $# -gt 0 ];  do
+    case $1 in
+    -d )
+        shift 1
+        DEFAULT_LOG_LEVEL=$1;
```

Comment:
LOG_LEVEL=$1;

DEFAULT_LOG_LEVEL is always constant. If user does not provide log level then it falls back to default log level.

Reply:
it's already handled at line 10: DEFAULT_LOG_LEVEL=${DEFAULT_LOG_LEVEL:-$INFO}  if user don't provide -d option the default is set to INFO